apps/valkyrie/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM bitwalker/alpine-elixir:1.9 as builder
```

Comment:
This probably needs to follow the pattern of the other apps where it is based upon the smartcitiesdata:build

Reply:
done